build.gradle

```diff
@@ -27,6 +34,10 @@ allprojects {
             url "https://dl.bintray.com/touchlab/Squeaky"
         }
         maven { url "https://jitpack.io" }
+        maven {
```

Comment:
Let's put google first, before jcenter. There was a recent issue (not sure if it's still happening) where a dependency was still being advertised in jcenter but was changed to only be available in google's artifact repo.

Reply:
Fixed this